doc/design/parallel-node-config.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Parallel SR-IOV configuration
+authors:
+  - e0ne
+reviewers:
+  - adrianchiris
+  - SchSeba
+creation-date: 18-07-2023
+last-updated: 18-07-2023
+---
+
+# Parallel SR-IOV configuration
+
+## Summary
+Allow SR-IOV Network Operator to configure more than one node at the moment.
+
+## Motivation
+SR-IOV Network Operator configures SR-IOV one node at a time and one nic at a same time. That means we’ll need to wait
+hours or even days to configure all NICs  on large cluster deployments. With multi-NICs deployments Operator configures
+NICs one by one on each node which  leads to a lot of unnecessary node drain calls during SR-IOV configuration. Also
```

Comment:
are we really saving up on drain calls ?

even currently sriov-network-config-daemon will configure all relevant nic in the node. it just does it serially.

Reply:
fixed